### DISCUSSION TITLE
return character in text field

### DISCUSSION THREAD

**Discussion by MKeaneIre26** [dbeaver/dbeaver#17202] [Q&A]
Hi, excuse the question but inexperienced in SQL, data export etc. 

I have a return character in a text field that is causing the field to split into two when I export - is there I way I can adjust the export setting in DBeaver to handle this ?

![db-return](https://user-images.githubusercontent.com/110247992/181795075-0c0d9b7a-186e-4803-aaa8-2d86c913351f.png)

**Comment by LonwoLonwo** [comment]
Hello @SFKing22 
What type of export do you use?
Different export types have different export settings.
